webr-build.sh

```diff
@@ -38,7 +38,7 @@ mv lib/* ${ROOT}/lib
 BIN="${ORIG}/repo/bin/emscripten/contrib/${R_VERSION}/"
 
 mkdir -p $BIN
-mv *.tgz $BIN
+mv *.tar.gz $BIN
```

Comment:
I thought that source packages were always .tar.gz and binary packages were always .tgz?

@gaborcsardi is that the case?

Reply:
Yes, macos packages are `.tgz` on CRAN, and the base are tooling also creates `.tgz` files, e.g. for `R CMD INSTALL --build`. Installation works from `.tar.gz` as well on macOS.

Reply:
I tried building an empty package on a Linux box and got a `.tar.gz` file,
```
$ R CMD INSTALL --build .
* installing to library '/home/georgestagg/R/x86_64-pc-linux-gnu-library/3.6'
* installing *source* package 'empty' ...
[...]
* creating tarball
packaged installation of 'empty' as 'empty_0.0.0.9000_R_x86_64-pc-linux-gnu.tar.gz'
* DONE (empty)
```

The format also includes extra platform information in the filename. This will make things trickier with our `webr::install()` script since it also [assumes the package filename is `packagename_version.tgz`](https://github.com/r-wasm/webr/blob/eb58fa6ebd2de671292165d36e013292e92b048d/packages/webr/R/install.R#L45).

Perhaps the `webr-build.sh` script should just rename the binary package into that expected format at the same time as moving the file into place, since we're already in control of the platform by putting our files under `bin/emscripten`.